C#/C#_Assignment1/C_Sharp_A1/Program.cs

```diff
@@ -0,0 +1,139 @@
+﻿//1.Write a C# program that performs all string operations like COMPARE, CONCAT, SUBSTRING, REVERSE using – 
+//a.	Manual Array operations for character array ‘char[]’
+//b.Built -in functions for ‘string’ variables
+
+namespace CsharpString
+{
+    using System;
+    using System.Text.RegularExpressions;
+    enum PopRank
+    {
+        India,
+        USA,
+        Japan,
+        France,
+        Australia,
+        England
+    }
+    class C_Sharp_A1
+    {
+        public static void Main(string[] args)
+        {
+            string str1 = "Luffy";
+            string str2 = "Shanks";
+            string str3 = "Shanks";
+            string concated = string.Concat(str1 + str2);
+            Boolean checkEqual = str1.Equals(str2);
+            Boolean checkEqual1 = str2.Equals(str3);
+            Console.WriteLine("Concated string is = " + concated);
+            Console.WriteLine("Is str1 equal to str 2? = " + checkEqual);
+            Console.WriteLine("Is str2 equal to str 3?  = " + checkEqual1);
+            Console.WriteLine("Reverse the string through a function we get = " + StringReverse.ReverseString("Rohan"));
+            Console.WriteLine("We can get substring by passing starting nad ending index in a function is = " + str1.Substring(0, 3));
+            
+            static string Rev(string inp)
+            {
+                char[] inpArr = inp.ToCharArray();
+                string reverse = String.Empty;
+                for (int i = inpArr.Length - 1; i > -1; i--)
+                {
+                    reverse += inpArr[i];
+                }
+                return reverse;
+            }
+            Console.WriteLine("Reverse the string using custom logic we get = " + Rev("Rohan"));
+            
+            static string Comp(string inp1, string inp2)
```

Comment:
comp ?  
function names should clearly state the functionality

same for Rev

Reply:
Those were shortforms for like compare & reverse. But yeah, i get your point they arent descriptive enough. Will correct them.

Reply:
Done with changes.